R/util_data_summaries.R

```diff
@@ -1,71 +1,63 @@
-###############################################################
+##########################################################################
 ###
 ###     util_data_summaries.R
-###     Used to summarize data.
 ###
-###     To summarize individual variable, call:
+###     The goal of the `ds` module is broadly to:
+###         * Make it fast and easy to build models and summarize data.
+###         * Standardize the data summaries we create for ourselves
+###         for and collaborators. 
 ###
-###     ds.summarize_by_column(    data , 
-###                     [func_list] , 
-###                     [codebook] 
-###                 )     
-###
-###     It is anticipated that other data summary functions
-###     will be added later.
 ###
 ###     Depends on util.R
 ###
-###############################################################
+##########################################################################
 
 # wrap ds.helper functions in an object to protect the namespace
-ds.helper <- list(
-    
-    mean = function(x){
-        if( util.is_vector_of_numbers(x) ){
-            return(mean(as.numeric(x), na.rm=TRUE))
-        }else{
-            return(NA)
-        }
-    }
-    ,
-    pct_blank = function(x){
-        mean(util.is_blank(x)) * 100
-    }
-    ,
-    sd = function(x){
-        sd(x, na.rm=TRUE)
-    }
-    ,
-    obs_min = function(x){
-        if( util.is_vector_of_numbers(x) ){
-            return(min(as.numeric(x),na.rm=T))
-        }else{
-            return(NA)   
-        }
-    }
-    ,
-    obs_max = function(x){
-        if( util.is_vector_of_numbers(x) ){
-            return(max(as.numeric(x),na.rm=T))
-        }else{
-            return(NA)   
-        }
+ds.helper <- list()
+
+ds.helper$mean <- function(x){
+    if( util.is_vector_of_numbers(x) ){
```

Comment:
The Wickham says: http://adv-r.had.co.nz/Style.html
PERTS style guide: https://docs.google.com/document/d/1nO0rFe3MHPDE6gwpHhzFH5PGBV8D5gqtuNXbBgtwL2M/edit

This line, and others like it, should be:

`if (util.is_vector_of_numbers(x)) {`